docs/variables.md

```diff
@@ -4,108 +4,106 @@ linkTitle: "Variable Substitutions"
 weight: 15
 ---
 -->
-# Variable Substitutions
+# Variable Substitutions Supported by `Tasks` and `Pipelines`
 
-This doc aggregates the complete set of variable substitions available
-in `Tasks` and `Pipelines`.
+This page documents the variable substitions supported by `Tasks` and `Pipelines`.
 
-## Variables Available in a Pipeline
+## Variables available in a `Pipeline`
 
 | Variable | Description |
 | -------- | ----------- |
-| `params.<param name>` | The value of the param at runtime. |
-| `tasks.<task name>.results.<result name>` | The value of a Task's result (**Note**: Affects Task ordering in a Pipeline!) |
+| `params.<param name>` | The value of the parameter at runtime. |
+| `tasks.<task name>.results.<result name>` | The value of the `Task's` result. Can alter `Task` execution order within a `Pipeline`.) |
 
-## Variables Available in a Task
+## Variables available in a `Task`
 
 | Variable | Description |
 | -------- | ----------- |
-| `params.<param name>` | The value of the param at runtime. |
+| `params.<param name>` | The value of the parameter at runtime. |
 | `resources.inputs.<resource name>.path` | The path to the input resource's directory. |
 | `resources.outputs.<resource name>.path` | The path to the output resource's directory. |
-| `results.<result name>.path` | The path to the file where a Task's result should be written. |
-| `workspaces.<workspace name>.path` | The path to the mounted workspace. |
-| `workspaces.<workspace name>.claim` | The name of the PersistentVolumeClaim used as a volume source for the workspace or empty string if a volume source other than PersistentVolumeClaim was used. |
-| `workspaces.<workspace name>.volume` | The name of the volume populating the workspace. |
-| `credentials.path` | The path to the location of credentials written by the `creds-init` init container. |
+| `results.<result name>.path` | The path to the file that stores the `Task's` result. |
```

Comment:
Ideally what we'd communicate here is that a Step in a Task needs to write a file at this path in order to get some data exposed as a result in the TaskRun. It's kind of a call-to-action, "Write a file here in your Step to expose its contents as a result in the TaskRun."

Maybe this already describes that and I'm just reading it with the wrong emphasis.  I think the problem might be that "The path to the file" is the focus of the sentence whereas ideally the action to be taken, Writing to the file at that path, would be the focus.  Does this make any sense?

Either way, not a blocker at all.

Reply:
No problem. Reworded to "The path to the file where the `Task` writes its results data." Let me know if that works.

Reply:
That's great, thankyou!